### DISCUSSION TITLE
Change to directory

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [goolord/alpha-nvim#52] [Q&A] [answered]
Does Alpha support something similar to `startify_change_to_dir` ?

**Comment by goolord** [comment, marked as answer]
not atm. this option could be trivial though, i guess. bookmarks can already do this. i didn't use this option when i used startify for files because it seems to me like it would change to a directory i don't want 90% of the time.

**Reply by goolord** [reply]
i have been really insanely busy for the past few weeks unfortunately and i don't see it letting up soon, but if i can get around to it i will

**Reply by LamprosPitsillos** [reply]
Glad to hear that but please prioritize the other stuff before this, whenever you feel like it! Thanks for considering it !

**Reply by LamprosPitsillos** [reply]
Solved in (unmerged as of now ) #143
